lua/rust-tools/inlay_hints.lua

```diff
@@ -4,7 +4,7 @@ local M = {}
 
 function M.new()
   M.namespace = vim.api.nvim_create_namespace("textDocument/inlayHints")
-  local self = setmetatable({ cache = {}, enabled = false }, { __index = M })
```

Comment:
I am not that versed in internals, so I have no idea why and how this state was used. All I know that I wasn't able to read it externally.